src/devices/GrovePi/Sensors/DhtSensor.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GrovePiDevice.Models;
+using Iot.Device.GrovePiDevice.Models;
```

Comment:
Build is failing because you have this using namespace twice.

Reply:
yes, correcting it